R/rdeephaven/R/aggregate_wrapper.R

```diff
@@ -17,69 +20,150 @@ Aggregation <- R6Class("Aggregation",
+#' @param cols String or list of strings denoting the column(s) to aggregate. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to aggregate all non-grouping columns, which is only valid in the `agg_all_by()` operation.
+#' @return Aggregation function to be used in `agg_by()` or `agg_all_by()`.
+#' @export
 agg_median <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
   return(Aggregation$new(INTERNAL_agg_median, "agg_median", cols=cols))
 }
 
+#' @description
+#' Creates a Variance aggregation that computes the variance of each column in `cols` for each aggregation group.
+#' @param cols String or list of strings denoting the column(s) to aggregate. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to aggregate all non-grouping columns, which is only valid in the `agg_all_by()` operation.
+#' @return Aggregation function to be used in `agg_by()` or `agg_all_by()`.
+#' @export
 agg_var <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
   return(Aggregation$new(INTERNAL_agg_var, "agg_var", cols=cols))
 }
 
+#' @description
+#' Creates a Standard Deviation aggregation that computes the standard deviation of each column in `cols`, for each aggregation group.
+#' @param cols String or list of strings denoting the column(s) to aggregate. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to aggregate all non-grouping columns, which is only valid in the `agg_all_by()` operation.
+#' @return Aggregation function to be used in `agg_by()` or `agg_all_by()`.
+#' @export
 agg_std <- function(cols = character()) {
   verify_string("cols", cols, FALSE)
   return(Aggregation$new(INTERNAL_agg_std, "agg_std", cols=cols))
 }
 
+#' @description
+#' Creates a Percentile aggregation that computes the given percentile of each column in `cols` for each aggregation group.
+#' @param percentile Numeric scalar between 0 and 1 denoting the percentile to compute.
+#' @param cols String or list of strings denoting the column(s) to aggregate. Can be renaming expressions, i.e. “new_col = col”.
+#' Default is to aggregate all non-grouping columns, which is only valid in the `agg_all_by()` operation.
+#' @return Aggregation function to be used in `agg_by()` or `agg_all_by()`.
+#' @export
 agg_percentile <- function(percentile, cols = character()) {
   verify_in_unit_interval("percentile", percentile, TRUE)
   verify_string("cols", cols, FALSE)
   return(Aggregation$new(INTERNAL_agg_percentile, "agg_percentile", percentile=percentile, cols=cols))
 }
 
-agg_count <- function(col) {
+#' @description
+#' Creates a Count aggregation that counts the number of rows in each aggregation group.
+#' Note that this operation is not supported in `agg_all_by()`.
+#' @param col String denoting the name of the new column to hold the counts of each aggregation group.
+#' Defaults to "n".
+#' @return Aggregation function to be used in `agg_by()`.
+#' @export
+agg_count <- function(col = "n") {
```

Comment:
I think this was changed.  Look through for the few places from the other PR that were modified.